### PR TITLE
Update post previews in filterable lists

### PR DESCRIPTION
**Pull request by kurtw**
This is in response to stakeholder feedback in the post previews.

## Additions

- `post_date_description` in FilterControls organism
- `show_preview_categories` in FilterControls organism

## Testing

- Go to a filterable list page
- Categories should be seen on the post previews and the date should say `Published <date>`
- Toggle the categories in the FilterControls
- Change the date description
 - Changes should be shown in the list

## Review

- @richaagarwal 
- @kave 
- @rosskarchner

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/organisms/post-preview.html [hunk 1]
```diff
@@ -43,32 +43,35 @@
...
-                            page_url,
-                            'post_slug meta-header_left',
-                            false, form_id) }}
-                {% endif %}
-            {% else %}
-                {% for cat in post.categories.all() %}
-                    {% if loop.index > 1 %}
-                        |
+            {% if controls.categories.show_preview_categories %}
+                {% if 'newsroom' in controls.page_type %}
```
Comment (richaagarwal) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: you could use `type` here since you define it above

**Comment by richaagarwal** [comment]
Other than a minor thing I left a comment on, looks good

**Comment by kave** [comment]
can you rebase this branch to remove unnecessary commits. thanks

**Comment by kave** [comment]
tested works looks good to me :+1:

**Pull request merged**